### PR TITLE
Feature/write post req

### PR DESCRIPTION
**Pull request by raynelle03**

### CODE REVIEW

File: iosHackthonApp/iosHackthonApp/Add Meal Screen/AddMealView.swift [hunk 1]
```diff
@@ -7,11 +7,28 @@
...
 import SwiftUI
+import shared
 
 struct AddMealView: View {
-    var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
-    }
+	var body: some View {
+		//TODO: Pass the actual values when UI is complete
+		let sdk = PostMealApi()
```
Comment (Gmiller290488) [inline iosHackthonApp/iosHackthonApp/Add Meal Screen/AddMealView.swift]: When you are updating this PR, could you pass the MealsSDK through to this view.
I know this is just a placeholder function just to check it works, but just so both of the screens are getting instantiated in the same way :-)

Comment (raynelle03) [reply iosHackthonApp/iosHackthonApp/Add Meal Screen/AddMealView.swift]: Yup..done!

**Review by kirstybutler — APPROVED** [review]

File: iosHackthonApp/iosHackthonApp/Add Meal Screen/AddMealView.swift [hunk 2]
```diff
@@ -7,15 +7,50 @@
...
 struct AddMealView: View {
-    var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
-    }
+
+	@ObservedObject private(set) var viewModel: ViewModel
+	
+	var body: some View {
+		//TODO: Pass the actual values when UI is complete
+		let sdk = MealApi()
```
Comment (Gmiller290488) [inline iosHackthonApp/iosHackthonApp/Add Meal Screen/AddMealView.swift:18]: You don't use this anymore

File: iosHackthonApp/iosHackthonApp/Add Meal Screen/AddMealView.swift [hunk 3]
```diff
@@ -7,15 +7,50 @@
...
+		Button("Add a meal") {
+			let meal = Meal(id: "\(sdk.getUUID())",
```
Comment (Gmiller290488) [inline iosHackthonApp/iosHackthonApp/Add Meal Screen/AddMealView.swift]: Could we switch this out to:
`let meal = Meal(id: "\(viewModel.sdk.getUUID())",`

and get rid of the `let sdk = MealApi()` line please

Comment (raynelle03) [reply iosHackthonApp/iosHackthonApp/Add Meal Screen/AddMealView.swift]: Done

File: iosHackthonApp/iosHackthonApp/Add Meal Screen/AddMealView.swift [hunk 4]
```diff
@@ -7,15 +7,49 @@
...
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
-    }
+
+	@ObservedObject private(set) var viewModel: ViewModel
+	
+	var body: some View {
+		//TODO: Pass the actual values when UI is complete
+		Button("Add a meal") {
+			let meal = Meal(id: "\(viewModel.sdk.getUUID())",
+											name: "Pizza",
```
Comment (dsmith42) [inline iosHackthonApp/iosHackthonApp/Add Meal Screen/AddMealView.swift:20]: The alignment seems a bit too far to the right to me.  Could we align them to `id: ...` above?

**Review by dsmith42 — COMMENTED** [review]
Looks good Raynelle 😀

**Review by Gmiller290488 — APPROVED** [review]

File: iosHackthonApp/iosHackthonApp/Add Meal Screen/AddMealView.swift (hunk 4 above)
Comment (raynelle03) [reply iosHackthonApp/iosHackthonApp/Add Meal Screen/AddMealView.swift:20]: ahh..I think that's just GitHub rending it that way...Looks fine in Xcode  :)

**Pull request merged**